dexmaker/src/test/java/com/google/dexmaker/stock/ProxyBuilderTest.java

```diff
@@ -879,4 +881,65 @@ public void setFakeResult(Object result) {
             fakeResult = result;
         }
     }
+
+    public static class TestOrderingClass {
+        public int returnsInt() {
+            return 0;
+        }
+
+        public String returnsString() {
```

Comment:
Overload one of these methods with some parameters to exercise more of your code